test/CMakeLists.txt

```diff
@@ -24,7 +24,8 @@ if (NOT TARGET gtest)
   add_subdirectory(googletest/googletest EXCLUDE_FROM_ALL)
```

Comment:
Can we be able to replace this with ExternalProjects_Add(...). I started experimenting this feature this week, but haven't succeeded yet, just putting that thought here.

Reply:
You are welcome to try it out. There isn't any specific advantage of that over the submodule I think

Reply:
Oh, thanks for the info. I just came across, just wanted to hear your comments. Looks like it may not be advantageous then.

Reply:
meshio is header only and gtest is for tests alone, so it's okay I guess

Reply:
Sure, that' ok, I asked this to more like I wanted to learn new features in cmake...